lib/drivers/basescan.js

```diff
@@ -0,0 +1,113 @@
+const Driver = require('../models/driver');
+const Supply = require('../models/supply');
+const Coin = require('../models/coin');
+const SupplyModifier = require('../models/supply-modifier');
+const { promisesMap } = require('../util');
+
+/**
+ * Basescan driver. Supports circulating and total supply for basechain
+ *
+ * @augments Driver
+ * @memberof Driver
+ */
+class Basescan extends Driver {
+  constructor(options) {
+    super({
+      timeout: 200, // 5 requests per second
+      supports: {
+        native: false,
+        balances: true,
+        nativeBalances: false,
+        tokens: true,
+        decimals: true, // Should this be required?
+        secret: true, // Should this be required?
+        blockchains: ['Basechain'],
```

Comment:
```suggestion
        blockchains: ['Base'],
```
Most sources I find, the blockchain is named Base or otherwise referred to as Base Chain, but that is just referencing the chain of Base.